<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cauhoithuonggap_form</class>
 <widget class="QMainWindow" name="cauhoithuonggap_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cauhoithuonggap_form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblcauhoithuonggap">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(154, 205, 244)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Câu hỏi thường gặp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/ic_return.png</normaloff>images/ic_return.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Làm sao để thay đổi đơn vị 
 tiền tệ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Ứng dụng có cảnh báo khi  
tôi chi tiêu vượt mức không?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Làm thế nào để liên hệ với 
 bộ phận hỗ trợ nếu gặp lỗi?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Ứng dụng có thể đề xuất cách cân 
bằng thu nhập và chi tiêu không?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Làm sao để đặt giới hạn chi tiêu 
 theo ngày hoặc tuần?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string> Tôi quên mật khẩu, làm sao 
 để đặt lại mật khẩu?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Làm sao để thay đổi đơn vị 
 tiền tệ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncauhoi8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 229, 238);</string>
    </property>
    <property name="text">
     <string>Làm thế nào để tắt/bật thông 
  báo từ ứng dụng?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblvande">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 200, 223);
border-radius: 10px
</string>
    </property>
    <property name="text">
     <string>     Vấn đề bạn cần hỗ trợ?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
